src/fiatconnect/fiatAccountSchemas/mobileMoney.ts

```diff
@@ -0,0 +1,73 @@
+import { FiatAccountSchema, FiatAccountType } from '@fiatconnect/fiatconnect-types'
+import { getObfuscatedAccountNumber } from 'src/fiatconnect'
+import { FiatAccountFormSchema } from 'src/fiatconnect/fiatAccountSchemas/types'
+import { FiatAccountSchemaCountryOverrides } from 'src/fiatconnect/types'
+import i18n from 'src/i18n'
+
+export const getMobileMoneySchema = (
+  implicitParams: {
+    country: string
+    fiatAccountType: FiatAccountType
+  },
+  countryOverrides?: FiatAccountSchemaCountryOverrides
+): FiatAccountFormSchema<FiatAccountSchema.MobileMoney> => {
+  const overrides = countryOverrides?.[implicitParams.country]?.[FiatAccountSchema.MobileMoney]
+  const mobileValidator = (input: string) => {
+    const regex = overrides?.mobile?.regex ? new RegExp(overrides.mobile.regex) : /^[0-9+]+$/
+    const isValid = regex.test(input) && input.length >= 4 && input.length <= 16
+    const errorMessageText = overrides?.mobile?.errorString
+      ? i18n.t(
+          `fiatAccountSchema.mobileMoney.mobile.${overrides.mobile.errorString}`,
+          overrides.mobile.errorParams
+        )
+      : i18n.t('fiatAccountSchema.mobileMoney.mobile.errorMessage')
+    return {
+      isValid,
+      errorMessage: isValid ? undefined : errorMessageText,
+    }
+  }
+
+  const mobileFormatter = (input: string) => {
+    if (input.length && input[0] != '+') {
+      return `+${input}`
+    } else if (input === '+') {
+      return ''
```

Comment:
This would mean that if the user starts typing with a plus, it would immediately be replaced with `''` right? Is that intentional? Also since we're automatically including the `+`, should there be some text indicating they should include their area code? Otherwise they could just type their local number right?